mod_reforged/hooks/msu.nut

```diff
@@ -61,6 +61,15 @@
 	}
 }
 
+
+::logInfo("Reforged::MSU -- adding ::MSU.Time.getSecondsRequiredToTravel");
+::MSU.Time.getSecondsRequiredToTravel <- function(_numTiles, _speed, _onRoadOnly)
+{
+	local speed = _speed * ::Const.World.MovementSettings.GlobalMult;
+	if (_onRoadOnly) speed = speed * ::Const.World.MovementSettings.RoadMult;
```

Comment:
Sounds like `onRoadOnly` should be an optional parameter that defaults to `false`?